pkg/vpcmodel/grouping.go

```diff
@@ -197,14 +197,18 @@ func (g *GroupConnLines) getGroupedEndpointsElems(grouped groupedEndpointsElems)
 }
 
 // same as the previous function, for groupedExternalNodesMap
-func (g *GroupConnLines) getGroupedExternalNodes(grouped groupedExternalNodes) *groupedExternalNodes {
+func (g *GroupConnLines) getGroupedExternalNodes(grouped []Node) *groupedExternalNodes {
+	res := make(groupedExternalNodes, len(grouped))
+	for i := range grouped {
+		res[i] = grouped[i].(*ExternalNetwork)
```

Comment:
Do all castings guaranteed to succeed?

Reply:
Replaced `Node` with `*ExternalNetwork` in more places where it can be used, so now the casting take place where it is clear that it should succeed (switch case checking `IsExternal()`).